### PR TITLE
Add ci.yaml for github workflows

### PR DESCRIPTION
**Pull request by jylinv0**
Add instrumentation test to github workflow

### CODE REVIEW

File: launcher/launcher.sh [hunk 1]
```diff
@@ -34,6 +34,9 @@ SED_SCRIPT=${TEST_TMPDIR}/runner_config.sed
 # enums.
 echo 's#\\*\"@!([^!]+)!@\\*\"#\1#g' > ${SED_SCRIPT}
 
+unset rlocation
+unset is_absolute
```
Comment (mithriltabby) [inline launcher/launcher.sh:42]: What's this doing?

Comment (jylinv0) [reply launcher/launcher.sh:42]: They're functions that created by bazel during test. The generation of sed script doesn't take care of function, so the script will be formatted incorrectly. The proper fix would be explicitly name which env vars to print in `printenv` or somehow handle functions properly during the generation.

Comment (mithriltabby) [reply launcher/launcher.sh:42]: Add a comment explaining this, or adjust the grep expression used to remove lines from the printenv output and add the comment there. It'd be very difficult to know ahead of time exactly which environment variables we want to expand.

Comment (jylinv0) [reply launcher/launcher.sh:42]: Added comments.

**Pull request closed without merging**